### ISSUE TITLE
Permission problems

### ISSUE THREAD

**Issue by aaronw2** [jlesage/docker-crashplan-pro#401]
It looks like your docker image defaults to uid 1000, gid 1000, which is a problem for me. Due to a lot of history, my uid is 500 and the user gid is 100. I would like the docker image to run with my uid (500) and with gid 2000, which I have assigned for use by crashplan since I have several TB of data backed up locally.

Right now there are a lot of permission issues. It cannot access much of my data (due to a mismatching uid) and cannot access the backup medium as well.

Is there any way to assign it to use these IDs?

**Comment by jlesage** [comment]
Yes, you need to adjust the `USER_ID` and `GROUP_ID` environment variables.
See:
https://github.com/jlesage/docker-crashplan-pro#environment-variables
https://github.com/jlesage/docker-crashplan-pro#usergroup-ids

**Comment by aaronw2** [comment]
Thanks.  It's now working and able to synchronize with my local backup drive. I created a new group for this.

**Comment by jlesage** [comment]
Great!
